Userland/Applications/Spreadsheet/SpreadsheetView.cpp

```diff
@@ -380,7 +380,7 @@ SpreadsheetView::SpreadsheetView(Sheet& sheet)
                 return;
 
             auto first_position = source_positions.take_first();
-            m_sheet->copy_cells(move(source_positions), move(target_positions), first_position);
```

Comment:
That variable name commit like...6 commits ago should just be merged with this one.